From 21920e95fb41f7b89244e3a3f3c16b412133b622 Mon Sep 17 00:00:00 2001
From: Min RK <benjaminrk@gmail.com>
Date: Fri, 24 Oct 2025 16:15:33 -0700
Subject: [PATCH] skip t_filters_parallel

seems to have weird problems and take forever on mpich
---
 testpar/Makefile.am | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/testpar/Makefile.am b/testpar/Makefile.am
index 4a8cb826f4..a6f7f988db 100644
--- a/testpar/Makefile.am
+++ b/testpar/Makefile.am
@@ -33,7 +33,7 @@ check_SCRIPTS = $(TEST_SCRIPT_PARA)
 
 # Test programs.  These are our main targets.
 #
-TEST_PROG_PARA=t_mpi t_bigio t_cache t_cache_image t_pread t_pshutdown t_prestart t_init_term t_pmulti_dset t_select_io_dset t_shapesame t_filters_parallel t_2Gio t_vfd
+TEST_PROG_PARA=t_mpi t_bigio t_cache t_cache_image t_pread t_pshutdown t_prestart t_init_term t_pmulti_dset t_select_io_dset t_shapesame t_2Gio t_vfd
 
 if SUBFILING_VFD_CONDITIONAL
     TEST_PROG_PARA += t_subfiling_vfd
-- 
2.49.0

